controllers/appwrapper_controller.go

```diff
@@ -177,25 +220,42 @@ func (r *AppWrapperReconciler) finalizeScalingDownMachines(ctx context.Context,
 				"deletionMessage", deletionMessage,
 			)
 			if err := r.deleteMachineSet(ctx, appwrapper); err != nil {
+				logger.Error(err, "Error deleting MachineSet for AppWrapper",
+					"appwrapper", appwrapper)
 				return err
 			}
 		}
-	} else {
+
+	case MachineTypeNodePool:
+		logger.Info(
+			"AppWrapper deleted, scaling down nodepool",
+			"appWrapper", appwrapper,
+			"deletionMessage", deletionMessage,
+		)
+		if _, err := r.deleteNodePool(ctx, appwrapper); err != nil {
+			logger.Error(err, "Error deleting NodePool for AppWrapper",
+				"appwrapper", appwrapper)
+			return err
+		}
+
+	case MachineTypeMachinePool:
 		logger.Info(
 			"AppWrapper deleted, scaling down machine pool",
 			"appWrapper", appwrapper,
 			"deletionMessage", deletionMessage,
 		)
 		if _, err := r.deleteMachinePool(ctx, appwrapper); err != nil {
+			logger.Error(err, "Error deleting MachinePool for AppWrapper",
+				"appwrapper", appwrapper)
```

Comment:
Why do we log the entire appwrapper?

Reply:
Hey Mark, 
Thanks for the review, this is done to fall in line with the structured logging format across components: https://github.com/project-codeflare/codeflare-operator/issues/113